.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         # the Node.js versions to build on
-        node-version: [12.x, 13.x, 14.x, 15.x, 16.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
Because this is a Verified Homebridge plugin I don't want to modify their template, would you mind reverting this change? 

See https://github.com/homebridge/homebridge-plugin-template/blob/master/.github/workflows/build.yml#L12

Reply:
Homebridge (and [verified plugins](https://github.com/homebridge/homebridge/wiki/Verified-Plugins)) support all current Active and Maintenance LTS releases of Node.js. Current supported versions of node.js are 14.x, 16.x, 18.x (note no odd versions). 

Reference: [https://github.com/homebridge/homebridge/wiki/How-To-Update-Node.js/](https://github.com/homebridge/homebridge/wiki/How-To-Update-Node.js/)

Reply:
Okay thanks for looking into that. Will accept then.